### PR TITLE
Add MTU to the plugin config

### PR DESCRIPTION
**Pull request by mogren**
*Issue #665*

*Description of changes:*
* The plugin needs to get the MTU from the config file
* Add documentation of log level config.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by mogren** [comment]
With `AWS_VPC_ENI_MTU="1500"`
```
> ifconfig | grep mtu
eth0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
eth1: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
eth2: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
eth3: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
lo: flags=73<UP,LOOPBACK,RUNNING>  mtu 65536
veth36109221769: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
veth01c9b2465d9: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
veth05d88c0bccc: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
veth07249681a7b: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
.
.
.
```

Without setting the environment variable
```
> ifconfig | grep mtu
eth0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 9001
eth1: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 9001
eth2: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 9001
eth3: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 9001
lo: flags=73<UP,LOOPBACK,RUNNING>  mtu 65536
veth02b26604dc0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 9001
veth0909d6b58ea: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 9001
veth0df9d65f2fc: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 9001
veth0f233f9a231: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 9001
.
.
.
```

File: README.md [hunk 1]
```diff
@@ -298,7 +310,7 @@ on `:61678/metrics`.
 
 Type: String
 
-Default: `eni`
+Default: `veth`
```
Comment (jaypipes) [inline README.md]: unrelated change? Also, I remember seeing some pushback on the whole eni -> veth rename patch from Calico. Would be good to undo this particular change I think.

File: pkg/networkutils/network.go [hunk 2]
```diff
@@ -247,6 +247,16 @@ func (n *linuxNetwork) SetupHostNetwork(vpcCIDR *net.IPNet, vpcCIDRs []*string,
 		}
 	}
 
+	if n.mtu != maximumMTU {
```
Comment (jaypipes) [inline pkg/networkutils/network.go]: Why guard the `LinkSetMTU()` call with this conditional? Why not just always set the MTU for the link? This conditional assumes that the link will have its default MTU set to jumbo frames, but if that changes in the future, this conditional will introduce a bug, right?

Comment (mogren) [reply pkg/networkutils/network.go]: Thanks, good catch.

File: plugins/routed-eni/cni.go [hunk 3]
```diff
@@ -67,6 +69,9 @@ type NetConf struct {
 	// veth device name. It should be no more than four characters, and
 	// defaults to 'eni'.
 	VethPrefix string `json:"vethPrefix"`
+
+	// MTU for eth0
+	Mtu string `json:"mtu"`
```
Comment (jaypipes) [inline plugins/routed-eni/cni.go]: Something I noticed... but golang naming practices generally encourage acronyms to be capitalized when exported, so this really should be "MTU", not "Mtu", like "CNIVersion" is capitalized above...

See more here: https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Comment (mogren) [reply plugins/routed-eni/cni.go]: I agree, should be all upper case.
